docs/font-inspector.html

```diff
@@ -231,6 +243,26 @@ <h1>Free Software</h1>
             element.innerHTML = '<dt>' + Object.keys(font.kerningPairs).length + ' Pairs</dt><dd>' + JSON.stringify(font.kerningPairs) + '</dd>';
         }
     }
+
+    if(font.tables.cpal) {
+        let markup = ' ';
+        const cpal = document.getElementById("cpal-table");
+        const dt = cpal.querySelector('dt');
```

Comment:
Why check for existing dt ? Can't we directly update the parent <dl> in any case ?

Reply:
I just checked: The headlines for alle tables we output are always there, but the content of the cell may be `"Undefined"`, so the `<dt>` might not always be there.